src/parsing.rs

```diff
@@ -76,7 +76,9 @@ impl<'tu> TryFrom<Entity<'tu>> for BaoStruct {
             })
             .collect::<Result<Vec<_>, BaoError>>()?;
 
-        let size = ty.get_sizeof().map_err(|_| BaoError::InvalidStructSize)?;
+        let size = ty.get_sizeof().map_err(|serr| BaoError::InvalidStructSize {
+            message: ty.get_display_name() +":"+ &*serr.to_string()
```

Comment:
You should prefer something like: `message: format!("{}:{}", ty.get_display_name(), serr)`